systemd/system/user-4045.slice.d/lowmem.conf

```diff
@@ -0,0 +1,2 @@
+[Slice]
+MemoryLimit=8M
```

Comment:
On old systemd (i.e. what we have now) you also need to add `MemoryAccounting=true`